backend/aks/deploy.sh

```diff
@@ -33,6 +35,9 @@ helm_init_client
 
 # ssh_public_key needs to be a file. Build it regardless of {ssh,gpg}-agent, or
 # forwarding of agents:
+eval "$(ssh-agent)"
```

Comment:
I'm not sure of this.
The idea was to consume whatever key was in the keychain. That way, your personal clusters are always accessible with your key, and the CI ones with a (yet to be created) shared key.

I would prefer to fail with a message on `ssh-add -L`, if there are no keys on the keychain, instead of always creating one.

Also, note the comment just before this line: it used to take whatever ssh key was in the keychain, regardless if the keychain is managed by ssh-agent or gpg-agent. Now, it will always use ssh-agent.

Reply:
created a new secret `ssh-key-ci` on concourse.
corresponding code in `cap-ci`: https://github.com/SUSE/cap/pull/28/files#diff-ddacb7f6f728e8d3719cd265216c9a1bR18